hivemq-edge/src/main/java/com/hivemq/configuration/entity/adapter/SouthboundMappingEntity.java

```diff
@@ -70,16 +73,12 @@ public SouthboundMappingEntity(
         return topicFilter;
     }
 
+    @Override
     public void validate(final @NotNull List<ValidationEvent> validationEvents) {
-        if (topicFilter == null || topicFilter.isEmpty()) {
-            validationEvents.add(new ValidationEventImpl(ValidationEvent.FATAL_ERROR, "topicFilter is missing", null));
-        }
-        if (tagName == null || tagName.isEmpty()) {
-            validationEvents.add(new ValidationEventImpl(ValidationEvent.FATAL_ERROR, "tagName is missing", null));
-        }
-        if (fromNorthSchema == null || fromNorthSchema.isEmpty()) {
-            validationEvents.add(new ValidationEventImpl(ValidationEvent.FATAL_ERROR, "fromNorthSchema is missing", null));
-        }
+        EntityValidatable.notEmpty(validationEvents, topicFilter, "topicFilter");
```

Comment:
if these methods were not static and instead were default, we would not need to prefix `EntityValidatable` as they are only used in the contest of implementors.
I know that static methods may be slightly quicker because their location is resolved at compile time, but in the grand scheme of things I would opt for readability. 
This is just a personal taste comment :).

Reply:
They are indeed all static methods.